lms/static/js/toggle_login_modal.js

```diff
@@ -92,7 +92,11 @@
                         $notice.show().html(notice);
             // This is for activating leanModal links that were in the notice. We should have a cleaner way of
             // allowing all dynamically added leanmodal links to work.
-                        $notice.find('a[rel*=leanModal]').leanModal({top: 120, overlay: 1, closeButton: '.close-modal', position: 'absolute'});
+                        $notice.find('a[rel*=leanModal]').each(function() {
+                            var $link = $(this),
+                                closeButton = $link.data('modalCloseButtonSelector') || '.close-modal';
```

Comment:
Nit: I don't think you need to set the $link variable since it's only used once. Saying closeButton = $(this).data is clear enough.